### PR TITLE
Set default port to 8150

### PR DESCRIPTION
**Pull request by kopiczko**
Towards https://github.com/giantswarm/roadmap/issues/725

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 ## [Unreleased]
 
+### Changed
+
+- Set the default port to 8150.
```
Comment (erkanerol) [inline CHANGELOG.md:12]: @kopiczko isn't there a consumer of this port? do we need to adapt somewhere else?

File: helm/net-exporter/templates/daemonset.yaml [hunk 2]
```diff
@@ -53,16 +53,17 @@ spec:
           {{- if (.Values.NetExporter.DNSCheck.TCP.Disabled) }}
           - "-disable-dns-tcp-check={{ .Values.NetExporter.DNSCheck.TCP.Disabled }}"
           {{- end }}
+          - "-port={{ .Values.port }}"
```
Comment (AverageMarcus) [inline helm/net-exporter/templates/daemonset.yaml:56]:
```suggestion
          - "-port={{ .Values.port }}"
        ports:
        - name: http
          port: {{ .Values.port }}
  	  containerPort: {{ .Values.port }}
```

Then the probes can use the named port and we have a visual indication of the port exposed

File: main.go [hunk 3]
```diff
@@ -148,6 +148,7 @@ func main() {
 	var exporter *exporterkit.Exporter
 	{
 		c := exporterkit.Config{
+			Address: "http://0.0.0.0:" + port,
```
[the remark this answers is not in the document]
Comment (kopiczko) [reply main.go:151]: Sure we can, but why?

File: CHANGELOG.md (hunk 1 above)
Comment (kopiczko) [reply CHANGELOG.md:12]: There isn't any service bound to this port known to me. If there is. It seems to work as I was testing it.

**Review by AverageMarcus — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (erkanerol) [reply CHANGELOG.md:12]: `net-exporter` service is backed by this port. As far as I see, helm templating will update it as well. However, that service has `giantswarm.io/monitoring: "true"` label and Prometheus scrapes metrics via this port. I checked whether we need to update somewhere for this port and found this line https://github.com/giantswarm/prometheus-meta-operator/blob/master/files/templates/scrapeconfigs/additional-scrape-configs.template.yaml#L451

Could you please check Prometheus config&service discovery after updating this port?

File: main.go (hunk 3 above)
[the remark this answers is not in the document]
Comment (whites11) [reply main.go:151]: I think it's better code, but hey your call

File: CHANGELOG.md (hunk 1 above)
Comment (kopiczko) [reply CHANGELOG.md:12]: I tested that multiple times.

**Comment by kopiczko** [comment]
We don't need that. For some reason I thought this is running in the host network. But it isn't. I only had to change the DNS service name.

There is still some value in this PR because it fixes the port setting  but I don't want to invest time into testing.

**Pull request closed without merging**